README.md

```diff
@@ -1,6 +1,6 @@
 # ⛓️🔗⛓️ Template for IBC enabled Solidity contracts
 
-This repo provides a starter project to build [IBC](https://github.com/cosmos/ibc) enabled Solidity contracts that connect rollups to one another Polymer Hub, through the [vIBC core contracts](https://github.com/open-ibc/vibc-core-smart-contracts).
+This [IBC app in Solidity template repo](https://github.com/open-ibc/ibc-app-solidity-template/tree/main) provides a starter project serves as a starting point for building multichain smart contracts that leverage Polymer Hub to connect across rollups through our [vIBC core contracts](https://github.com/open-ibc/vibc-core-smart-contracts).
```

Comment:
we can prob just get rid of this link altogether, doesn't make sense to link to the repo the readme is hosted in :p

Reply:
oh this is also link our docs as quickstart